[2024-11-13T05:25:16.827+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-13T05:25:16.839+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: email_pipeline_dag.process_and_save_to_postgres scheduled__2024-11-13T05:20:00+00:00 [queued]>
[2024-11-13T05:25:16.844+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: email_pipeline_dag.process_and_save_to_postgres scheduled__2024-11-13T05:20:00+00:00 [queued]>
[2024-11-13T05:25:16.844+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-11-13T05:25:16.861+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): process_and_save_to_postgres> on 2024-11-13 05:20:00+00:00
[2024-11-13T05:25:16.900+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=490) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-13T05:25:16.905+0000] {standard_task_runner.py:72} INFO - Started process 492 to run task
[2024-11-13T05:25:16.906+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'email_pipeline_dag', 'process_and_save_to_postgres', 'scheduled__2024-11-13T05:20:00+00:00', '--job-id', '26', '--raw', '--subdir', 'DAGS_FOLDER/email_pipeline_dag.py', '--cfg-path', '/tmp/tmpect2nk32']
[2024-11-13T05:25:16.917+0000] {standard_task_runner.py:105} INFO - Job 26: Subtask process_and_save_to_postgres
[2024-11-13T05:25:17.065+0000] {task_command.py:467} INFO - Running <TaskInstance: email_pipeline_dag.process_and_save_to_postgres scheduled__2024-11-13T05:20:00+00:00 [running]> on host a91f7ad55282
[2024-11-13T05:25:17.143+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='email_pipeline_dag' AIRFLOW_CTX_TASK_ID='process_and_save_to_postgres' AIRFLOW_CTX_EXECUTION_DATE='2024-11-13T05:20:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-13T05:20:00+00:00'
[2024-11-13T05:25:17.145+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-13T05:25:19.083+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-11-13T05:25:19.103+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-13T05:25:19.103+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=email_pipeline_dag, task_id=process_and_save_to_postgres, run_id=scheduled__2024-11-13T05:20:00+00:00, execution_date=20241113T052000, start_date=20241113T052516, end_date=20241113T052519
[2024-11-13T05:25:19.165+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-13T05:25:19.216+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-13T05:25:19.219+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
